src/utils.js

```diff
@@ -403,6 +403,10 @@ class Utils extends VueUtils {
 		};
 		return resolver(schema);
 	}
+	static extractUDPParams(process) {
+		let [id, ...namespace] = process.split('@');
+		return [id, namespace.join('@')]
```

Comment:
```suggestion
		return [id, namespace.join('@')];
```

Interesting how "stupid" such a code looks in JS while in other languages it's just a matter of providing a limit to split.

Reply:
These little differences between programming languages are always tricky. In this case, the code indeed looks rather "stupid" to just make a simple split work.